tools/cli/commands/changelog.js

```diff
@@ -5,6 +5,7 @@ import process from 'process';
 import { fileURLToPath } from 'url';
 import chalk from 'chalk';
 import inquirer from 'inquirer';
+import autoComplete from 'inquirer-autocomplete-standalone';
```

Comment:
I had looked at `inquirer-autocomplete-prompt` but saw it was only receiving critical bug fixes now and they recommended `inquirer-autocomplete-standalone` - plus we didn't match the version of `inquirer` required by the package, so I went with the standalone.

However doing a cursory look at `enquirer` looks like it might be the better move indeed. It has autocomplete built in and the way it works seems more similar to the existing implementation, so switching might be easier. Let me take a look at that before we move forward here.

Reply:
https://github.com/Automattic/jetpack/pull/34403